### DISCUSSION TITLE
UTM parameters with FCM

### DISCUSSION THREAD

**Discussion by NorbertSanpruch** [invertase/react-native-firebase#6963] [Q&A]
Hi!

I have a problem with UTM parameters tracking with firebase cloud messaging.

I want to see how many sessions come from a mobile push notification. I am not interested in `notification_open` event but rather `session_start` with correct UTM parameters (from what I understand this event is responsible for traffic acquisition in GA). I have my own server that sends push notifications using `firebase-admin`. I pass UTM parameters in the notification payload but these are not automatically handled and I do not know how to do it manually.

I have deep links configured as well and the traffic from there is correctly propagated (deep links have attached UTM parameters) without adding extra code and I can see it in GA. I am not using firebase dynamic links and not planning to do that.

I can not find documentation for such case.

Do you have any idea how can I achieve "session channel group statistics" for mobile push notifications?

**Comment by MateoCoiro** [comment]
Hi, i am having the same issue. Do you have any updates on this?

**Comment by coyksdev** [comment]
Hi, I also save the same issue. Do you have a solution on this?
